### ISSUE TITLE
Are pings sent only via email at the moment?

### ISSUE THREAD

**Issue by 3ruce** [OrkoHunter/ping-me#6]
Or can a mobile number be specified also?

**Comment by OrkoHunter** [comment]
Hey @3ruce, The plan right now is to ping the user on their linux/osx/windows, their phone app (android/ios/windows) and any other "smart" device like smart watch. A text message will also be enabled, but that will cost some amount obviously.

PS. Thanks for showing interest. The library is extensively under process as of now.

**Comment by 3ruce** [comment]
Is there a phone app that can be downloaded at the moment? I guess a phone app makes the text message less important...

**Comment by OrkoHunter** [comment]
The phone app will be there after `ping-me` is done with desktop notifications. But it will be soon.

**Comment by 3ruce** [comment]
cool

**Comment by 3ruce** [comment]
Have you thought about making this a paid for service too i.e. command line for open source users and pingme.com for a hosted version with a small monthly fee/freemium where peope just send an email/fill out a line in the app to get the same benefits?

**Comment by OrkoHunter** [comment]
I mostly do free and open source work. But I'll consider this option. Thanks.

**Comment by OrkoHunter** [comment]
However, can you elaborate the idea of pingme.com a little bit more? That'd be great.

**Comment by 3ruce** [comment]
So the basic principles of the project is that people set really simple _calendarless_ reminders to suit their schedule and their either write in 'pure language' as per the readme file, or they set simple times and dates. I'm not aware of anythng this simple (although it may well exist!) either on the command line or elsewhere (perhaps it's in here https://play.google.com/store/search?q=reminder%20app&c=apps&hl=en)...

The simplicity of emailing or writing a reminder is very convenient, so perhaps some people would pay for a web hosted version where they have a standard email address they can use and/or an app?

Does that make any sense at all?

**Comment by OrkoHunter** [comment]
Yeah sure. But I was hoping to monetize only some of the features, like sms and more than 10 reminders at a time. I'll think about it.

**Comment by 3ruce** [comment]
It's the 'pure language' option that might hook people, plus a simple time/date picker as an alternative (like all the others do at the moment)

**Comment by OrkoHunter** [comment]
Yes. The natural language processing is a very good part of the project. Plus, people who don't want to leave terminal sometimes, would like it,

**Comment by 3ruce** [comment]
Exactly...

**Comment by harshcrop** [comment]
can you explain  me this so, I can start work on it??

**Comment by 3ruce** [comment]
Sure, what part needs explaining?
